Internals/Extensions/API/Modules/Extension.js

```diff
@@ -0,0 +1,68 @@
+const sizeof = require("object-sizeof");
+const {
+	Errors: {
+		Error: GABError,
+	},
+} = require("../../../index");
+let rawExtensionDocument;
```

Comment:
Is never set

Reply:
Extensions/API is in-progress, there's no point to reviewing it at this stage.